components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -11,14 +11,13 @@
   } from "@commons/methods/component";
   import { weekdays } from "@commons/methods/datetime";
   import { NotificationMode } from "@commons/enums/Scheduler";
-  // TODO: switch for local development
-  // import "@nylas/components-availability";
-  // import "../../availability";
+
+  import { DefaultCustomFields } from "@commons/constants/custom-fields";
+
   import "../../availability/src/Availability.svelte";
-  // import "@nylas/components-scheduler";
-  // import "../../scheduler";
```

Comment:
Figured we were past due to remove these; current method works both in prod and local